plumbing/format/idxfile/idxfile.go

```diff
@@ -11,7 +11,7 @@ var (
 	idxHeader = []byte{255, 't', 'O', 'c'}
 )
 
-// An Idxfile represents an idx file in memory.
+// Idxfile in memory representation of a idx file
```

Comment:
The sentence doesn't have a verb, change it to: Idxfile is the in memory representation of an idx file.
End the sentence with a full stop.